spec/features/idv/in_person_spec.rb

```diff
@@ -229,6 +229,11 @@
       "#{t('date.day_names')[6]}: #{t('in_person_proofing.body.barcode.retail_hours_closed')}",
     )
 
+    # expire enrollment while still signed in
+    InPersonEnrollment.where(enrollment_code: enrollment_code).last.update(status: :expired)
+    visit root_path
```

Comment:
@NavaTim the example you shared with me from the cancel spec is doing this so I opted to just use this line instead of including the entire conditional. The problem I'm having is that when I run this test it fails b/c the `current_path` is evaluating to `/account`. If i change the value  I'm checking against to '/account' then `current_path` evaluates to the welcome step but the test fails as it say's it expected the path to the ready to verify page.  
If a user clicks the `start over` link in this scenario it should take them to the `root_url`. 
Testing this out locally when you click the link you arrive at the account page and have to manually enter verify which then brings you to the welcome step. 
Testing w/ sinatra running same thing happens but you can click a link on your account page to arrive at the welcome step.

Reply:
i think i misinterpreted the error - it's the next expect that is failing. But my test still isn't working as expected.

Reply:
The important part to test is that the IdV process directs the user to the welcome step instead of the barcode step **_when they navigate back to IdV_**. Everything else about reaching specific pages seems like it's out of scope.

Reply:
I think that for the use case here, it is acceptable (until/unless we receive contrary input from design) to even direct the user back to the `/accounts` page if they attempt to use the original emailed link.

Reply:
the problem is the user is just directed to the account page and not the welcome step when running the test

Reply:
If you can keep the original SP context, then I think it's better to send them back to the welcome page, but if not then I think the account page is alright so long as they can later navigate to the verify page.